<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Homomorphic Encryption Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>输入工资 1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="salary1Input"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>输入工资 2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="salary2Input"/>
    </item>
    <item>
     <widget class="QPushButton" name="encryptButton">
      <property name="text">
       <string>加密数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="compareButton">
      <property name="text">
       <string>比较工资</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>状态: 等待输入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>结果: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ciphertextLabel">
      <property name="text">
       <string>加密密文:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="ciphertextOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>